### PR TITLE
TCD/JDJ/RULE19-25

### PR DESCRIPTION
**Pull request by JacksonJ-KC**

### CODE REVIEW

File: rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_25.json [hunk 1]
```diff
@@ -0,0 +1,458 @@
...
+            "ruleset_reference": "Table G3.1 Section 4 & G3.1.2.4",
+            "rule_description": "Schedules for HVAC fans that provide outdoor air for ventilation shall run continuously whenever spaces are occupied in the baseline design.",
+            "applicable_rmr": "Baseline RMR",
+            "rule_assertion": "=",
+            "comparison_value": "Expected Value",
+            "primary_rule": "Yes",
+            "schema_version": "0.0.23"
+        },
+        "rmr_transformations": {
+            "baseline": {
```
Comment (yunjoonjung-PNNL) [inline rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_25.json:19]: Could you also add ```user``` and ```proposed```? We need these for the ```get_proposed_hvac_modeled_with_virtual_cooling``` and ```get_proposed_hvac_modeled_with_virtual_heating``` functions.

**Comment by yunjoonjung-PNNL** [comment]
Could you also add ```proposed```? This is because ```proposed``` is required for the ```get_proposed_hvac_modeled_with_virtual_cooling``` and ```get_proposed_hvac_modeled_with_virtual_heating``` functions.

**Comment by JacksonJ-KC** [comment]
> Could you also add `proposed`? This is because `proposed` is required for the `get_proposed_hvac_modeled_with_virtual_cooling` and `get_proposed_hvac_modeled_with_virtual_heating` functions.

Yes, sorry for the hold up! I will be correcting the TCDs for 19-10, 19-14, 19-20, and 19-25 this weekend.

**Comment by yunjoonjung-PNNL** [comment]
> > Could you also add `proposed`? This is because `proposed` is required for the `get_proposed_hvac_modeled_with_virtual_cooling` and `get_proposed_hvac_modeled_with_virtual_heating` functions.
> 
> Yes, sorry for the hold up! I will be correcting the TCDs for 19-10, 19-14, 19-20, and 19-25 this weekend.

No problem at all! Take you time. Once this is ready, I'll use it for the test.

File: rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_25.json (hunk 1 above)
Comment (JacksonJ-KC) [reply rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_25.json:269]: Added

**Review by yunjoonjung-PNNL — APPROVED** [review]
Looks great to me. Thanks!

**Pull request merged**